README.md

```diff
@@ -477,6 +477,12 @@ All statements above will return false if used with `===`
 
 🔗 [**Read More: Refactoring!**](/sections/testingandquality/refactoring.md)
 
+## ![✔] 4.10 Run tests in parallel
+
```

Comment:
Consider starting the section with the "so what" - why is it important, there are many sources on the testing feedback loop and why providing the developer with almost-immediate feedback is valuable